populate/src/main/resources/seed/i18n/dev/languageTexts.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
I like the amount of scoping you did on the keys for these.  they communicate where they are/what they do, while still being concise and readable